README.md

```diff
@@ -10,6 +10,7 @@ Docker images used within scalableminds :whale:
 |---|  ---  |    ---    |
 | [alpine](alpine)   | [![](https://images.microbadger.com/badges/image/scalableminds/various:alpine__master.svg)](https://microbadger.com/images/scalableminds/various:alpine__master)   | [scalableminds/various:alpine__…](https://hub.docker.com/r/scalableminds/various/tags/) |
 | [cronut](cronut)   | [![](https://images.microbadger.com/badges/image/scalableminds/cronut:master.svg)](https://microbadger.com/images/scalableminds/cronut:master)   | [scalableminds/cronut](https://hub.docker.com/r/scalableminds/cronut/tags/) |
+ [puppeteer](puppeteer)   | [![](https://images.microbadger.com/badges/image/scalableminds/puppeteer:master.svg)](https://microbadger.com/images/scalableminds/puppeteer:master)   | [scalableminds/puppeteer](https://hub.docker.com/r/scalableminds/puppeteer/tags/) |
```

Comment:
As long as we don't have a specific repo for `puppeteer` this image will be pushed to the dockerhub `various` repo, like
[scalableminds/various:puppeteer__add-puppeteer__73](https://hub.docker.com/r/scalableminds/various/tags/)
@normanrz Do you want to add a dockerhub repo? Or we wait first if we use it for a while, and then add one.

Reply:
@normanrz Ping :)

Reply:
I added the `puppeteer` repo